### PR TITLE
🐛✅ PRTL-1233 Range facets turn into Term facets when query changes

### PR DESCRIPTION
**Pull request by cheapsteak**

### CODE REVIEW

**Comment by cy** [comment]
looks like has the strange effect of, it 2 range filters are added, the first one is lost 🤔

**Comment by cheapsteak** [comment]
That happens also on relay
Merge and make new ticket?
Less merge conflicts

**Comment by cy** [comment]
🎫 https://jira.opensciencedatacloud.org/browse/PRTL-1236

**Comment by cheapsteak** [comment]
range filters seems to be broken currently on relay branch 
hesitant to merge this in since this could be blamed as the culprit

**Comment by cy** [comment]
broken in what way, wrong data returning or filter set wrong?

**Comment by cheapsteak** [comment]
Is working for me now as well, weird
Adding a range filter was resulting in a javascript error complaining something about ctx

File: modules/node_modules/@ncigdc/utils/filters/index.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ export const replaceFilters: TMergeFilters = (q, ctxq) => {
   const merged = {
     op: 'and',
     content: ctxq.content.reduce((acc, x) => {
```
Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/utils/filters/index.js]: is this `x` supposed to be `ctx`?

Comment (cheapsteak) [inline modules/node_modules/@ncigdc/utils/filters/index.js]: I think you're right

Comment (cheapsteak) [inline modules/node_modules/@ncigdc/utils/filters/index.js]: or the other `ctx`s should be `x`

Comment (cheapsteak) [inline modules/node_modules/@ncigdc/utils/filters/index.js]: PR open
thanks for catching this
https://github.com/NCI-GDC/portal-ui/pull/1708/files

**Pull request merged**